### PR TITLE
[kie-issues#248] Add string concatenation examples.

### PR DESCRIPTION
**Pull request by baldimir**
@tarilabs could you please review?

### CODE REVIEW

**Review by tarilabs — APPROVED** [review]
https://github.com/kiegroup/dmn-feel-handbook/blob/87b311da56b0619d3f7a56e77aad29caa9667324/test.java#L3

I believe it will complain you are not using latest version, so you'll likely have to bump it

**Comment by baldimir** [comment]
Version bumped.

File: source/index.html.md [hunk 1]
```diff
@@ -460,6 +460,17 @@ FEEL supports three-valued logic (3VL) semantic. You can use `and` and `or` as t
 Please notice that <code>and</code> is evaluated before <code>or</code>.
 </aside>
 
+## String concatenation
+
+> Examples
+
+```FEEL,commented
```
Comment (tarilabs) [inline source/index.html.md:467]:
```suggestion
```FEEL
```

**Review by tarilabs — APPROVED** [review]

**Comment by tarilabs** [comment]
I believe I had 2/3 ways to ensure proper integration tests:

1 expression per line with the resulting being a boolean `true` --> ` ```FEEL `
1 expression per line and line ending with comment with `//` and the arrow, with a valid FEEL value, to check for equality

and just plain text

**Pull request merged**